addons/web/static/src/core/l10n/dates.js

```diff
@@ -398,6 +407,20 @@ export function formatDuration(seconds, showFullDuration) {
     return durationSplit.slice(0, numberOfValuesToDisplay).join(",");
 }
 
+/**
+ * Formats a DateTime object to a `'W'W kkkk` datetime string, in the user's locale settings.
+ *
+ * @param {NullableDateTime} value
+ * @returns {string}
+ */
+export function formatLocalWeekYear(value) {
+    if (!value) {
+        return "";
+    }
+    const { year, week } = getLocalYearAndWeek(value);
+    return `W${week} ${year}`;
```

Comment:
```suggestion
    return `${_t("Week")[0]}${week} ${year}`;
```
We could localize this harder by taking the first letter of `_t("Week")`, but I'm only fluent in languages where "Week" starts with "W", so no idea whether "W52" already makes sense in e.g. Mandarin or French.